models/user.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    minlength: 2,
+    maxlength: 30,
+    required: true,
+  },
+  about: {
+    type: String,
+    minlength: 2,
+    maxlength: 30,
+    required: true,
+  },
+  avatar: {
+    type: String,
+    required: true,
+    validate: {
+      validator(value) {
+        return validator.isUrl(value);
```

Comment:
Нужно исправить: при сохранении пользователя происходит ошибка валидации.
```json
{
    "message": "Что-то пошло не так",
    "err": "user validation failed: avatar: validator.isUrl is not a function"
}
```
```suggestion
        return validator.isURL(value);
```